PubSub.Test/Unit/Core/HttpSubscriberNotifierServiceTest.cs

```diff
@@ -2,14 +2,20 @@
 using Moq;
 using Moq.Protected;
 using System.Net;
-using Microsoft.Extensions.Configuration;
+using System.Text.Json;
+using Newtonsoft.Json;
```

Comment:
Unused reference, also JsonSerializer using won't be needed after removing the Newtonsoft reference.

Make sure Newtsonsoft.Json package is even needed in the project